superset/assets/src/dashboard/util/charts/getEffectiveExtraFilters.js

```diff
@@ -54,7 +54,22 @@ const getFilter = (col, op, val) => {
   }
 }
 
-const valueToString = (value) => value ? value.toString() : value;
+const valueToStringErr = (value) => {
+  console.log("Error: [Prototype-less object]/ [null] encountered for getEffectiveExtraFilters");
```

Comment:
I am not sure syntax but there must be some way to use logger to log this at Info level

Reply:
Okay, I'll check.